crates/bevy_render_2d/src/mesh2d/color_material.wgsl

```diff
@@ -56,9 +56,8 @@ fn alpha_discard(material: ColorMaterial, output_color: vec4<f32>) -> vec4<f32>
         // NOTE: If rendering as opaque, alpha should be ignored so set to 1.0
         color.a = 1.0;
     }
-#ifdef MAY_DISCARD
-    else if alpha_mode == COLOR_MATERIAL_FLAGS_ALPHA_MODE_MASK {
-       if color.a >= material.alpha_cutoff {
+#ifdef MAY_DISCARD else if alpha_mode == COLOR_MATERIAL_FLAGS_ALPHA_MODE_MASK {
```

Comment:
what happened here?

Reply:
vscode seems to have started doing some sort of overzealous auto format on saves, I'll revert it